docs/docs/guide/10-debug.md

```diff
@@ -1,27 +1,26 @@
 ---
-sidebar_position: 13
-description: Blockchain Debug
+description: Debugging your Cosmos SDK blockchain
 ---
 
-# Blockchain Debug
+# Debugging a chain
 
-Ignite chain debug command can help you to find issues during development.
-It uses [Delve](https://github.com/go-delve/delve) debugger which enables you to
+Ignite chain debug command can help you find issues during development. It uses
+[Delve](https://github.com/go-delve/delve) debugger which enables you to
 interact with your blockchain app by controlling the execution of the process,
 evaluating variables, and providing information of thread / goroutine state, CPU
 register state and more.
 
 ## Debug Command
 
 The debug command requires that the blockchain app binary is build with
-debugging support by removing optimizations and inlining.
-A debug binary is built by default by the `ignite chain serve` command or can
-optionally be created using the `--debug` flag when running `ignite chain init`
-or `ignite chain build` sub-commands.
+debugging support by removing optimizations and inlining. A debug binary is
+built by default by the `ignite chain serve` command or can optionally be
+created using the `--debug` flag when running `ignite chain init` or `ignite
+chain build` sub-commands.
 
 To start a debugging session in the terminal run:
 
-```bash
+```
```

Comment:
I didn't know that the `bash` syntax didn't work on single line commands. Maybe we should keep them for consistency?

Reply:
`bash` just makes strange choices on what to highlight.

<img width="791" alt="Screenshot 2023-01-05 at 16 21 40" src="https://user-images.githubusercontent.com/332151/210789697-a4fbbcff-344d-4a87-95ed-95072d120acd.png">